### PR TITLE
Document optional MQTT cover options

### PR DESCRIPTION
**Pull request by thecode**
## Proposed change

Make all payloads optional to allow cover that can be just open (e.g. gates that self close after some time)

Example entry for `configuration.yaml`:

```yaml
cover:
  - platform: mqtt
    payload_open: "on"
    payload_close: 
    payload_stop: "on"
```

or, via auto discovery:

```json
{
  "cover": [
    {
      "platform": "mqtt",
      "payload_open": "on",
      "payload_close": null,
      "payload_stop": "on"
    }
  ]
}
```

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [X] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/50579
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

File: source/_integrations/cover.mqtt.markdown [hunk 1]
```diff
@@ -422,6 +422,45 @@ cover:
...
+
+{% raw %}
+
+```yaml
+# Example configuration.yaml entry
+cover:
+  - platform: mqtt
+    payload_open: "on"
+    payload_close: 
+    payload_stop: "on
```
Comment (maxcanna) [inline source/_integrations/cover.mqtt.markdown]: You missed a quote here 🙂

**Review by maxcanna — CHANGES_REQUESTED** [review]
There's a small typo in the example

File: source/_integrations/cover.mqtt.markdown (hunk 1 above)
Comment (frenck) [reply source/_integrations/cover.mqtt.markdown]:
```suggestion
    payload_stop: "on"
```

**Review by frenck — APPROVED** [review]

**Pull request merged**